syncapi/storage/postgres/syncserver.go

```diff
@@ -86,6 +88,11 @@ func NewDatabase(dbProperties *config.DatabaseOptions) (*SyncServerDatasource, e
 	if err != nil {
 		return nil, err
 	}
+	m := sqlutil.NewMigrations()
+	deltas.LoadFixSequences(m)
+	if err = m.RunDeltas(d.db, dbProperties); err != nil {
```

Comment:
It's a bit more complicated - https://github.com/matrix-org/dendrite/blob/master/roomserver/storage/postgres/storage.go#L45

Reply:
Oh this doesn't add a new column?

Reply:
Nope, which is mostly why I thought I could probably get away with it as it is.